featuretools/primitives/api.py

```diff
@@ -1,4 +1,5 @@
 # flake8: noqa
+from .rolling_primitive_utils import apply_roll_with_offset_gap, roll_series_with_gap
```

Comment:
So I actually want to have a discussion on this. I'm curious as to why we have an `api.py` file instead of just doing an `__all__` in `__init__.py` since this is essentially the same thing and we could get rid of the flake8 ignores. My intention with this was to just make the imports simpler but if this is only for exposing user facing functions then I'd be okay with removing.

Reply:
I'm not sure if this is what @thehomebrewnerd is alluding to, but if these functions aren't public, should we keep the `_` at the begining?